### ISSUE TITLE
Tor build is broken

### ISSUE THREAD

**Issue by Dor1s** [google/oss-fuzz#1033]
https://oss-fuzz-build-logs.storage.googleapis.com/index.html

```
Step #3:   CC       src/or/src_or_libtor_testing_a-circuitmux_ewma.o
Step #3: src/common/memarea.c:325:1: error: expected identifier or '('
Step #3: memarea_drop_all(memarea_t *area)
Step #3: ^
Step #3: ./src/common/memarea.h:13:3: note: expanded from macro 'memarea_drop_all'
Step #3:   do {                                          \
Step #3:   ^
Step #3: src/common/memarea.c:325:1: error: expected identifier or '('
Step #3: ./src/common/memarea.h:16:5: note: expanded from macro 'memarea_drop_all'
Step #3:   } while (0)
Step #3:     ^
Step #3: 2 errors generated.
Step #3: Makefile:6205: recipe for target 'src/common/src_common_libor_testing_a-memarea.o' failed
Step #3: make: *** [src/common/src_common_libor_testing_a-memarea.o] Error 1
Step #3: make: *** Waiting for unfinished jobs....
Finished Step #3
ERROR
```

//cc @nmathewson

**Comment by nmathewson** [comment]
Thanks for the heads-up!  This looks like a breakage in the --disable-memory-sentinels configuration option, caused by the recently merged https://trac.torproject.org/projects/tor/ticket/24337.  Should be fixed with commit 58e8094816cb7d2dbc6ecc71892b667e968d9e2d

**Comment by Dor1s** [comment]
Confirmed, thanks!
